lms/models/lti_params.py

```diff
@@ -123,3 +123,18 @@ def _get_key(data: dict, data_path: List[str]):
         value = value[path_item]
 
     return value
+
+
+def _get_lti_params(request):
```

Comment:
We want to use LTIParams to get the product across LTI versions.

Currently LTIParams is tied to context which makes it tricky to use outside a view.